pyomo/contrib/mindtpy/initialization.py

```diff
@@ -42,7 +44,10 @@ def MindtPy_initialize_master(solve_data, config):
     if config.use_dual:
         m.dual.deactivate()
 
-    if config.strategy == 'OA':
+    if config.init_strategy == 'feas_pump':
+        MindtPy.MindtPy_linear_cuts.fp_no_cycling_cuts = ConstraintList(
+            doc='Cycling cuts')
```

Comment:
There is a mismatch in the naming between `cycling` and `no_cycling` here. Let's settle with `orthogonality` cuts.